src/sdk-reference/js/6/extend-sdk/index.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: sdk.html.hbs
+title: Extends the SDK
+description: Extends the SDK
+order: 410
+---
+
+# Extends the SDK with a custom SDK controller
+
+It is possible to extend the SDK's functionality by adding new controllers.  
+
+These controllers generally correspond to [custom controllers created in a plugin]({{ site_base_path }}plugins/1/essentials/controllers) but not necessarily. Thus, it is possible to use the actions of its plugin in the SDK in the same way as the other actions of the Kuzzle API.  
+
+## Define a custom SDK controller
+
+A custom SDK controller is a class inheriting from the [BaseController]({{ site_base_path }}sdk-reference/js/6/base-controller) class and defining methods that correspond to API actions.  
+
+This class is exposed alongside the other class of the SDK module.  
```

Comment:
```suggestion
This class is exposed alongside the other classes of the SDK module.  
```

Also, I'm not sure about the exact meaning of that sentence. Though if I interpret it correctly, then you may want to replace it with something like that: `After it has been added using Kuzzle.use<link>, a custom controller can be accessed like any other SDK controllers`.

Reply:
I mean that the `BaseController` class is exposed like the `Kuzzle` or `WebSocket` class when you do a `require('kuzzle-sdk')`)